libpromises/eval_context.c

```diff
@@ -3484,6 +3484,47 @@ bool EvalContextIsIgnoringLocks(const EvalContext *ctx)
     return ctx->ignore_locks;
 }
 
+StringSet *ClassesMatchingLocalRecursive(const EvalContext *ctx, const char* regex, const Rlist *tags, bool first_only, size_t stack_index)
+{
+  assert(ctx != NULL);
```

Comment:
Indents should be four spaces

Reply:
I know! I tried running clang-format yesterday and it mangled the entire file horribly. So was about to just clang-format my changes and then got distracted. I am a bit surprised we don't have a CI tasks/github action for checking style :(